custom-metrics-autoscaling/direct-to-sd/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Copyright 2018 Google Inc.
+
+FROM golang:1.7.3
+ADD . /go/src/direct-to-sd
+RUN go get cloud.google.com/go/compute/metadata
```

Comment:
replace with `go get ./...`

Reply:
can you also verify this image runs correctly. I'm a bit skeptical since first FROM and second FROM use debian and alpine respectively, and they have different libc.

Reply:
I rebuild and tested both images, they run correctly.

What's the advantage of using `go get ./...` here?